File: Inciante/Cálculo Simples.py
peca1,npeca1,vpeca1 = map(float,input().split())
peca2,npeca2,vpeca2 = map(float,input().split())
valor = npeca1*vpeca1 + npeca2*vpeca2
print("VALOR A PAGAR: R$", "%.2f"%valor)

File: Iniciante C/Cálculo Simples.c
#include <stdio.h>
int main(){
    int a,b,c,d;
    double w,z,conta;
    scanf("%d%d%lf",&a,&b,&w);
    scanf("%d%d%lf",&c,&d,&z);
    conta = (b*w)+(d*z);
    printf("VALOR A PAGAR: R$ %.2lf\n",conta);
    return 0;
}